packages/node_modules/@webex/plugin-meetings/src/common/logs/request.js

```diff
@@ -37,6 +37,8 @@ export default class LogsRequest {
     }
     catch (error) {
       LoggerProxy.logger.error('Logs:request#uploadLogs --> uploading user logs failed', error);
+
+      return Promise.reject(error);
```

Comment:
```suggestion
      throw error;
```
Since this is an async function, we don't have to use promises here and just throw the error. Not a blocker on the PR though.